File: Recursion/Fibonacci/fibonacci.js
function fabonacci(n){
        if(n==0||n==1)
        {
            return n;
        }
        else
        {
            return fabonacci(n-1) + fabonacci(n-2);
        }
}
const ps = require("prompt-sync");
const prompt = ps();
let user;
while(true)
{
    user = parseInt(prompt("Enter Number: "));
    if(user>=0)
    {
        break;
    }
    console.log("Number can not be zero");

}
const result = fabonacci(user);
console.log(result);

File: Recursion/Fibonacci/fibonacci.java
import java.util.Scanner;

class Main{
    public static int fibonacci(int n)
    {
        if(n==0 || n==1)
        {
            return n;
        }
        else return fibonacci(n-1) + fibonacci(n-2);

    }
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int user;
        while(true)
        {
            System.out.println("Enter Number: ");
            user = scanner.nextInt();
            if(user>=0) break;
            System.out.println("Number can not be  zero");
        }
        System.out.println(fibonacci(user));
    }


}